src/main/java/com/boot/amazon/controller/ReviewController.java

```diff
@@ -14,15 +25,43 @@
 public class ReviewController {
     private final ReviewService reviewService;
     private final ReviewMapper reviewMapper;
+    private final UserService userService;
+    private final ProductSevice productSevice;
 
-    public ReviewController(ReviewService reviewService, ReviewMapper reviewMapper) {
+    public ReviewController(ReviewService reviewService,
+                            ReviewMapper reviewMapper, UserService userService,
+                            ProductSevice productSevice) {
         this.reviewService = reviewService;
         this.reviewMapper = reviewMapper;
+        this.userService = userService;
+        this.productSevice = productSevice;
     }
 
-    @GetMapping("/top-words")
+    @GetMapping("/admin/top-words")
     public ReviewRequestDto getWords(@RequestParam(defaultValue = "1000") int limit) {
         List<String> words = reviewService.getTopMostFrequentWordsInReview(limit);
         return reviewMapper.map(words);
     }
+
+    @DeleteMapping("/admin/delete")
+    public void deleteReview(@RequestParam Long reviewId) {
```

Comment:
because reviewId it's the resource identifier which we want to delete it's better to pass it as @PathVariable